pkg/zson/docs/spec.md

```diff
@@ -358,20 +353,20 @@ record(
 )
 ```
 The special characters that must be escaped if they appear within a value are:
-* newline (\n)
-* backslash (\\)
-* the separator character (usually \t)
-* the set_separator character inside of set and vector values (usually ,)
-* unset_field (usually -) if it appears as a value not be interpreted as "unset",
-
-Similarly, a set with no values must be specified  by the empty_field string (usually "(empty)")
-to distinguish it from a set with a single value "" and this must be escaped if it
-is a single-element set with the value "(empty)", i.e., escaped as "\(empty)".
-
-When processing legacy values, a column of type string named "_path" is
-inserted into value provided a #path directive previously appeared in the
-stream.  The contents of this _path field is set to the string value indicted
-in the #path directive and all the other columns are shifted one space to
+* newline (`\n`)
+* backslash (`\`)
+* the `#separator` character (usually `\t`)
+* the `#set_separator` character inside of set and vector values (usually `,`)
+* `#unset_field` (usually `-`) if it appears as a value not be interpreted as "unset",
+
+Similarly, a `set` with no values must be specified by the `#empty_field` string (usually `(empty)`)
+to distinguish it from a `set` with a single value that's a zero-length string, and this must be escaped if it
+is a single-element set with the value `(empty)`, i.e., escaped as `\x28empty)`.
```

Comment:
The single backslash that was here before was being swallowed during rendering, so clearly we needed to do something to show this example of escaping. Since "legacy" is basically synonymous with the way Zeek ASCII TSV logs are output today (yes?) I opted to show the hex-escaped version since that's what Zeek does by default at the moment.

Reply:
There's consensus on `\x28`.